evaluation/evaluation.py

```
@@ -197,6 +200,96 @@ async def aquery(self, *args, **kwargs):
+        if isinstance(qk, str):
+            return qk
+        return None
+
+    def _is_no_answer(self, response: Any) -> bool:
+        try:
+            text = getattr(response, "response", None)
+            if text is None:
+                text = str(response)
+            text = (text or "").strip()
+            no_text = text in (NO_ANSWER_REFERENCE, NO_ANSWER_REFERENCE_PLACEHOLDER)
+        except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)